cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -3470,6 +3498,203 @@ if(ARROW_WITH_GRPC)
   endif()
 endif()
 
+# ----------------------------------------------------------------------
+# GCS and dependencies
+
+macro(build_crc32c_once)
+  if(NOT TARGET crc32c_ep)
+    message(STATUS "Building crc32c from source")
+    # Build crc32c
+    set(CRC32C_PREFIX "${CMAKE_CURRENT_BINARY_DIR}/crc32c_ep-install")
+    set(CRC32C_CMAKE_ARGS
+        ${EP_COMMON_CMAKE_ARGS}
+        -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+        "-DCMAKE_CXX_FLAGS=${EP_CXX_FLAGS}"
```

Comment:
Can we remove them because `EP_COMMON_CMAKE_ARGS` include them?

```suggestion
```

Reply:
Done.